.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,33 @@
-
-<!-- ⚠️ Please select either this section... ⚠️ -->
-### Editing an existing topic or collection
-
-I'm suggesting these edits to an existing topic or collection:
-- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
-- [ ] Content (and my changes are in `index.md`)
-
-> Please replace this line with an explanation of why you think these changes should be made.
-
-<!-- ⚠️ ... or this section ⚠️ -->
-### Curating a new topic or collection
-
-- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
-- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
-- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs
-
-> Please replace this line with an explanation of why you think this topic or collection should be curated.
-
----------------------------------------------------------------------
-
-**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
+### Thank you for contributing! Please confirm this pull request meets the following requirements:
+
+- [ ] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
+- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).
+
+### Which change are you proposing?
+
+  - [ ] Suggesting edits to an existing topic or collection
+  - [ ] Curating a new topic or collection
+
+---------------------------------------------------------------------
+
+<!-- ⚠️ Please select either this section... ⚠️ -->
+### Editing an existing topic or collection
+
+I'm suggesting these edits to an existing topic or collection:
+- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
+- [ ] Content (and my changes are in `index.md`)
+
+> Please replace this line with an explanation of why you think these changes should be made.
+
+<!-- ⚠️ ... or this section ⚠️ -->
+### Curating a new topic or collection
+
+- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
+- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
+- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs
+
+> Please replace this line with an explanation of why you think this topic or collection should be curated.
+
+---------------------------------------------------------------------
+
+**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
```

Comment:
Looks like this is still different than what is on master. I believe if you accept this suggestion is should fix it up.

```suggestion
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [ ] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

<!-- ⚠️ Please select either this section... ⚠️ -->
### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

<!-- ⚠️ ... or this section ⚠️ -->
### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

<!-- ⚠️ ... or this section ⚠️ -->
### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
```